webapp/www/faq/hybrid_labeling.html

```diff
@@ -153,6 +159,16 @@ <h4>
                         The Diary screen has not changed at all. None of the new features (e.g., the yellow labels, the confirm button, only seeing trips you need to label) work on the Diary screen. So we recommend you do your labeling on the Label screen!
                     </h4>
                     <p></p>
+                    <h3>
+                        <strong>I love algorithms! Can I have more details?</strong>
+                    </h3>
+                    <h4>
+                        We group all your data so that trips with the same start and end location are in the same bin. We then look at the labels for the trips in each bin and how often they occur. If a trip has (shared_ride,work,drove_alone) labels 60% of the time, and (drove_alone,work,shared_ride) 40% of the time, we will display the highest probability label - e.g. (shared_ride,work,drove_alone) as a "yellow" "To Label". If it occurs 100% of the time, though, after the first few times, we don't even need to put it into "To Label" because we are pretty sure that is right.
+                        But what if you used to carpool to work consistently, but now you started riding your spiffy new ebike? To detect such changes, we will periodically go from the usual "relaxed" mode to an "intensive" mode, where all your trips will show up in "To Label" for a week.
```

Comment:
```suggestion
                        But what if you used to carpool to work consistently, but now you started riding your spiffy new eBike? To detect such changes, we will periodically go from the usual "relaxed" mode to an "intensive" mode, where all your trips will show up in "To Label" for a week.
```
Nitpick: I've been using `e-bike`; this website seems to sometimes use `e-bike` and sometimes use `eBike`  — we should standardize on something but it doesn't seem to be `ebike`.